### PR TITLE
Issue 31 dl normalisation

### PR DESCRIPTION
**Pull request by Echsecutor**
Added Ralphs GS1 URI Normaliser to convert all matching uris to canonical digital link (DL) form

### CODE REVIEW

**Comment by Echsecutor** [comment]
@RalphTro could you check at least some of the prehashstrings changes in https://github.com/RalphTro/epcis-event-hash-generator/pull/45/commits/0099dc1b51281bb28d9f761bed097c307c8f7570 to verify that the normalisation works as expected?

I have converted your examples to `tests/dl_normaliser_test.py` maybe you could also have a look there if the URIs are properly converted.

**Comment by RalphTro** [comment]
Hi @Echsecutor ,
All IDs in 0099dc1 which I checked look good. Thanks a lot!

Did you already push the new file (tests/dl_normaliser_test.py)? Happy to have a look at them, too.

File: tests/dl_normaliser_test.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    assert normaliser('http://example.de/abc/9780345418913') is None
+
+
+def test_incorrect_syntax():
+    assert normaliser('http://example.de/01/97803') is None
+    assert normaliser('https://id.gs1.org/8006/04012345123456/22/ABCD/10/XYZ') is None
+    # the following must not throw
+    assert normaliser('42') is None
+    assert normaliser(42) is None
+    assert normaliser('Hello World!') is None
```
Comment (Echsecutor) [inline tests/dl_normaliser_test.py:129]: @RalphTro here

Comment (RalphTro) [reply tests/dl_normaliser_test.py]: Hi @Echsecutor , apart from the slight adjustment (see below: upui EPC URI), we can merge this from my POV!

**Comment by Echsecutor** [comment]
@RalphTro :+1:  thanks!

**Pull request merged**